presto-main/src/main/java/io/prestosql/util/CIDR.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.util;
+
+import java.net.Inet4Address;
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.StringTokenizer;
+
+public abstract class CIDR
```

Comment:
Is this class your code or copied from somewhere? If the latter, please include a code comment mentioning the source.  See for ex `io.prestosql.spi.block.IntArrayList`

Reply:
These class (CIDR, CIDR4 & CIDR6) are copied from Netty as David suggested. I'll add comment that.